azure-pipelines.yml

```diff
@@ -146,7 +154,7 @@ stages:
       - task: NuGetToolInstaller@1
         displayName: "Install NuGet"
 
-      - powershell: .\scripts\build\set-azp-project-version-variable.ps1
+      - template: set-azp-variables-steps.yml
```

Comment:
YML template file is relative to the outer template from @commonTempaltes. 
Having file in our repo produced error:
>/stage-with-burgr-notifications.yml@commonTemplates: File /azp-variables-steps.yml not found in repository https://dev.azure.com/sonarsource/DotNetTeam%20Project/_git/pipelines-yaml-templates branch refs/tags/v1.0.11 version 0b8ce2a631db4d551d8194bc472d2887069ba0a9.

The steps template is now here and redesigned to be reusable.
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_git/pipelines-yaml-templates?path=%2Fset-azp-variables-steps.yml